mobile-app/app/screens/AppNavigator/screens/Portfolio/components/BalanceTextV2.tsx

```diff
@@ -0,0 +1,17 @@
+import { ThemedProps, ThemedTextV2 } from '@components/themed'
+import { TextProps } from '@components'
+
+interface BalanceTextProps {
+  symbol?: string
+  value: string
+}
+
+export function BalanceTextV2 ({ symbol, value, ...otherProps }: BalanceTextProps & ThemedProps & TextProps): JSX.Element {
+  return (
+    <ThemedTextV2
+      {...otherProps}
+    >
+      {value}
```

Comment:
what happen to displaying the hidden balance text on BalanceText(v1)?

Reply:
Not needed, only for the portfolio page

Reply:
For the existing BalanceText component in the portfolio page

Reply:
Will replace HAHA